water/Templates/styles.css

```diff
@@ -0,0 +1,17 @@
+.purple_button button.bk.bk-btn.bk-btn-default {
+    background-color: purple;
+    border-color: purple;
+    color: white;
+}
+
+.blue_button button.bk.bk-btn.bk-btn-default {
+    background-color: #34c3eb;
+    border-color: #34c3eb;
+    color: white;
+}
+
+.green_button button.bk.bk-btn.bk-btn-default {
+    background-color: #07db1c;
+    border-color: #07db1c;
+    color: white;
+}
```

Comment:
It might be time we started documenting our colour choices somehow so that we keep a consistent set between css and Python.

Reply:
Add comments